prow/jobs/test-infra/ko-build.yaml

```diff
@@ -0,0 +1,64 @@
+presubmits:
+  kyma-project/test-infra:
+    - name: pull-test-infra-ko-build
+      decorate: true
+      cluster: untrusted-workload
+      annotations:
+        owner: neighbors
+        description: Builds go-based toolkit in one go
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pull-test-infra-ko-build"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+      spec:
+        containers:
+          - image: europe-docker.pkg.dev/kyma-project/prod/testimages/buildpack-go:v20230717-e09b0fee
+            env:
+              - name: KO_DOCKER_REPO
+                value: europe-docker.pkg.dev/kyma-project/prod/test-infra
+            command: [ "ko" ]
+            args:
+              - resolve
+              - -f
+              - .koapps.yaml
+              - --push=false
+              - --base-import-paths
+              - -j4
```

Comment:
```suggestion
              -j4
```

Reply:
It's a list of arguments :D `-j4` is a shorthand for `--jobs=4`

Reply:
ok :D